src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.ts

```diff
@@ -217,8 +217,8 @@ export class SmbClusterFormComponent extends CdForm implements OnInit {
   }
 
   multiSelector(event: any, field: 'label' | 'hosts') {
-    if (field === PLACEMENT.host) this.selectedLabels = event.map((label: any) => label.content);
-    else this.selectedHosts = event.map((host: any) => host.content);
+    if (field === PLACEMENT.host) this.selectedHosts = event.map((label: any) => label.content);
```

Comment:
```suggestion
    if (field === PLACEMENT.host) this.selectedHosts = event.map((host: any) => host.content);
```

Reply:
Aaagh... sorry about that. Seems like sometimes I just can not read, will update it